schema/compose-spec.json

```diff
@@ -13,7 +13,6 @@
 
     "name": {
       "type": "string",
-      "pattern": "^[a-z0-9][a-z0-9_-]*$",
```

Comment:
Do we have this validation somewhere else in the source code?

Reply:
yes, [NormalizeProjectName](https://github.com/compose-spec/compose-go/blob/main/loader/loader.go#L669-L673)